curvefs/src/metaserver/storage/memory_storage.cpp

```diff
@@ -0,0 +1,51 @@
+/*
+ *  Copyright (c) 2022 NetEase Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/*
+ * Project: Curve
+ * Date: 2022-02-18
+ * Author: Jingli Chen (Wine93)
+ */
+
+#include <string>
+#include <memory>
+
+#include "curvefs/src/metaserver/storage/utils.h"
+#include "curvefs/src/metaserver/storage/memory_storage.h"
+
+namespace curvefs {
+namespace metaserver {
+namespace storage {
+
+MemoryStorage::MemoryStorage(StorageOptions options)
+    : options_(options) {}
+
+bool MemoryStorage::GetStatistics(StorageStatistics* statistics) {
+    statistics->MaxMemoryBytes = options_.MaxMemoryBytes;
+    statistics->MaxDiskQuotaBytes = options_.MaxDiskQuotaBytes;
+
+    uint64_t vmRSS;
+    if (!GetProcMemory(&vmRSS)) {
+        return false;
+    }
+    statistics->MemoryUsageBytes = vmRSS;
+    statistics->DiskUsageBytes = 0;
```

Comment:
raft also use disk, should we pass disk usage?

Reply:
> raft also use disk, should we pass disk usage?

done.